sdks/python/container/base_image_requirements.txt

```diff
@@ -28,46 +28,46 @@ avro-python3==1.8.2;python_version>="3.4"
 fastavro==0.21.4
```

Comment:
Check out for other newer dependencies in setup.py. (For example pymongo == 3.8.0 )